content/en/docs/reference/access-authn-authz/authentication.md

```diff
@@ -1144,7 +1144,7 @@ If specified, `clientKeyData` and `clientCertificateData` must both must be pres
 {{< /tabs >}}
 
 Optionally, the response can include the expiry of the credential formatted as a
-RFC3339 timestamp. Presence or absence of an expiry has the following impact:
+[RFC 3339](https://datatracker.ietf.org/doc/html/rfc3339) timestamp. Presence or absence of an expiry has the following impact:
```

Comment:
```suggestion
[RFC 3339](https://datatracker.ietf.org/doc/html/rfc3339) timestamp.
Presence or absence of an expiry has the following impact:
```

Reply:
Thanks, @tengqm for taking look at this. I will update it soon!

Reply:
Hi @tengqm, I have addressed the comment, PTAL!